packages/cli/internal/pkg/cli/context/manager.go

```diff
@@ -78,6 +83,21 @@ type ProgressResult struct {
 var displayProgressBar = cdk.DisplayProgressBar
 var showExecution = cdk.ShowExecution
 
+func (m *Manager) getEnvironmentVars() []string {
+	var environmentVars []string
+
+	for imageName := range m.imageRefs {
+		environmentVars = append(environmentVars,
+			fmt.Sprintf("ECR_%s_ACCOUNT_ID=%s", imageName, m.imageRefs[environment.WesImageKey].RegistryId),
+			fmt.Sprintf("ECR_%s_REGION=%s", imageName, m.imageRefs[environment.WesImageKey].Region),
+			fmt.Sprintf("ECR_%s_TAG=%s", imageName, m.imageRefs[environment.WesImageKey].ImageTag),
+			fmt.Sprintf("ECR_%s_REPOSITORY=%s", imageName, m.imageRefs[environment.WesImageKey].RepositoryName),
```

Comment:
Why are these always the WesImageKey?

Reply:
Good catch, I've also updated the underlying images so that our tests don't always use "WES"